.scalafmt.conf

```
@@ -1,4 +1,4 @@
-version = 2.0.0-RC7
+version = 2.0.0-RC8
```

Can't this be `2.0.0` now?
```suggestion
version = 2.0.0
```

---

I encountered an issue when I tried that. Let me retry to see what it was.

---

@raboof this is the error if you also update the .scalafmt.conf file, which makes sense, these are different - the conf file and the sbt plugin, e.g. https://github.com/scalameta/sbt-scalafmt/blob/master/.scalafmt.conf#L1

```
[error] failed to resolve Scalafmt version '2.0.3': .../akka/.scalafmt.conf
```

I think it is fine for `plugins.sbt` with [the latest release version](https://github.com/scalameta/sbt-scalafmt/releases)